Ginger/Ginger/UserControlsLib/UCListView/UcListView.xaml.cs

```diff
@@ -177,7 +177,7 @@ public IObservableList DataSourceList
 
                     mObjList = value;
 
-                    filteredView = CollectionViewSource.GetDefaultView(mObjList);
```

Comment:
Please test all POM functionalities including updating existing POM when filter is applied

Reply:
Testing complete